__tests__/app.test.js

```diff
@@ -184,6 +184,19 @@ describe.only('POST/api/reviews/:review_id/comments', () => {
         })
     });
     //error handling
+    test('can take a send request with extra properties and ignore them as long as valid info is contained', () => {
```

Comment:
please dont add changes to previous tickets to current ones it makes the branches and history redundant, we are trying to make it so that each branch only has the changes that are needed for that ticket, adding in amendments can add confusion and make it more difficult to review your code